### ISSUE TITLE
Feature request: Password reset is not exposed through an API

### ISSUE THREAD

**Issue by c-prompt** [matomo-org/matomo#14238]
updating the password in the SQL table would work 👍 Otherwise only chance would be to send the admin password unfortunately. 

In general it sounds like it's a problem that the password reset is not exposed through an API. Maybe create an issue for this? (although we would likely not work on this right away and would hope for some community contribution I suppose)

_Originally posted by @tsteur in https://github.com/matomo-org/matomo/issues/14099#issuecomment-474550637_

**Comment by mikkeschiren** [comment]
Yes, agree on that it should be in the API, I could be interested in looking into this as a contribution - as another contribution - I added the possibility to reset a users password in the plugin https://plugins.matomo.org/UserConsole#resetpassword

**Comment by Berbe** [comment]
@mikkeschiren The `UserConsole` plugin seems to have vanished from the plugins marketplace?
